File: bernardo/Problema0049/problema0049.py
# Entrada de valores
razao = float(input("Digite a razão da PG: "))
termo_1 = float(input("Digite o primeiro termo: "))
posicao_n = float(input("Digite a posição desejada: "))

# Cálculo do termo da PG
termo_n = termo_1 * (razao ** (posicao_n-1))

# Saída 
print(f"O termo{int(posicao_n)} é: {termo_n}")

File: bernardo/Problema0049/problema0049.java
import java.util.Scanner;

public class problema0049 {
    static Scanner input = new Scanner(System.in);

    public static void main(String[] args) {

        // Declaração de variáveis
        float razao;
        float termo1;
        float posicaoN;
        float termoN;

        // Entrada de valores
        System.out.println("Digite a razão da PG: ");
        razao = Float.valueOf(input.nextLine());
        System.out.println("Digite o primeiro termo: ");
        termo1 = Float.valueOf(input.nextLine());
        System.out.println("Digite a posição desejada: ");
        posicaoN = Float.valueOf(input.nextLine());

        // Cálculo do volume da lata
        termoN = (float) (termo1 * Math.pow(razao, posicaoN - 1));

        // Saída
        System.out.println("O termo" + posicaoN + " é: " + termoN);
    }
}
